x-pack/plugins/cloud_security_posture/public/common/navigation/translations.ts

```diff
@@ -7,18 +7,18 @@
 
 import { i18n } from '@kbn/i18n';
 
-export const CLOUD_POSTURE = i18n.translate('xpack.csp.navigation.cloudPosture', {
+export const CLOUD_POSTURE = i18n.translate('xpack.csp.navigation.cloudPostureBreadcrumbLabel', {
```

Comment:
i'm acutally not 100% sure about the next suggestion but those might be `Link` instead of `Label`. maybe even `xPageLinkLabel`